react/components/Button/index.js

```diff
@@ -236,6 +257,14 @@ Button.propTypes = {
   collapseLeft: PropTypes.bool,
   /** Cancels out right padding */
   collapseRight: PropTypes.bool,
+  /** */
```

Comment:
theese could be hidden, since it's just for button group purposes.
```suggestion
  /** @ignore */
```